src/bindings/Worker.resi

```diff
@@ -13,16 +13,16 @@ module Make: (Config: Config) =>
   include Config
 
   module App: {
-    let postMessage: (worker, fromApp) => unit
```

Comment:
Without these changes I was getting type errors. I posted about it on the forum: https://forum.rescript-lang.org/t/regarding-src-bindings-worker-res-in-the-rescript-lang-org-repo/2004/2